pkg/common/cns-lib/volume/manager.go

```diff
@@ -178,13 +178,18 @@ func (m *defaultManager) CreateVolume(ctx context.Context, spec *cnstypes.CnsVol
 			log.Errorf("CNS CreateVolume failed from vCenter %q with err: %v", m.virtualCenter.Config.Host, err)
 			return nil, err
 		}
-		var taskDetails createVolumeTaskDetails
-		// Store the task details and task object expiration time in volumeTaskMap
-		taskDetails.Lock()
-		taskDetails.task = task
-		taskDetails.expirationTime = time.Now().Add(time.Hour * time.Duration(defaultOpsExpirationTimeInHours))
-		volumeTaskMap[spec.Name] = &taskDetails
-		taskDetails.Unlock()
+		blockBackingDetails, ok := spec.BackingObjectDetails.(*cnstypes.CnsBlockBackingDetails)
```

Comment:
`2.0` supports file volume. Can we take care of static file volume provisioning?

Reply:
Sure! done